File: app/src/main/java/com/robotsandpencils/androiddaggerpracticemvp/lobby/LobbyViewModule.java
package com.robotsandpencils.androiddaggerpracticemvp.lobby;

import dagger.Binds;
import dagger.Module;

/**
 * Created by pwray on 2017-11-01.
 */
@Module
public abstract class LobbyViewModule {

    @Binds
    abstract LobbyGreetingContract.LobbyView provideLobbyView(LobbyActivity lobbyActivity);
}

File: app/src/main/java/com/robotsandpencils/androiddaggerpracticemvp/lobby/LobbyViewModule.kt
package com.robotsandpencils.androiddaggerpracticemvp.lobby

import dagger.Binds
import dagger.Module

/**
 * Created by pwray on 2017-11-01.
 */
@Module
abstract class LobbyViewModule {

    @Binds
    internal abstract fun provideLobbyView(lobbyActivity: LobbyActivity): LobbyGreetingContract.LobbyView
}
